FirebaseRealtimeDBHelper/FirebaseRealtimeDBHelper.cs

```diff
@@ -169,4 +179,44 @@ public FirebaseResponse Delete(string rootnode)
             return new FirebaseRequest(HttpMethod.Delete, rootnode).Execute();
         }
     }
+
+    class AccessAuthentication
+    {
+        public static string JWTToken { get; set; }
```

Comment:
Hi @shameeralibaig ,

Do use camel casing for method arguments and local variables.

Reply:
updated the above casing